### PR TITLE
Fix outdated reference to player._shardscollected

### LINKED ISSUE #595: [BUG REPORT] Unable to sell treasures for shards
Submitter: Andrew Goulding
I just started up the game again after playing from yesterday, as I booted it up it's now saying I have a shard case after not having one yesterday. All my shard upgrades are gone, and I cannot sell my underground treasures for shards. I have tried exporting my save & clearing my cookies and reimporting the save and it still doesn't seem to work.

### PR DESCRIPTION
**Pull request by nathan-c-jones**
Closes #595

### CODE REVIEW

**Comment by Ishadijcks** [comment]
Hi, thank you for your first contribution!

The fix seems to work, nice catch.

I have 2 small suggestions for future pull requests:
- Descriptive titles are usually more searchable, it should explain what the pull request does instead of what issue number it refers to, that is for the body.
- If you mention an issue, such as "Closes #number", it will automatically close the related issue.

I have changed these 2 nitpicky things for you :)

**Comment by nathan-c-jones** [comment]
Understood, thanks for the tips!

File: src/scripts/underground/Underground.ts [hunk 1]
```diff
@@ -171,7 +171,7 @@ class Underground {
             default:
                 const type = item.valueType.charAt(0).toUpperCase() + item.valueType.slice(1); //Capitalizes string
                 const typeNum = PokemonType[type];
-                player._shardsCollected[typeNum](player._shardsCollected[typeNum]() + GameConstants.PLATE_VALUE);
+                App.game.shards.shardWallet[typeNum] += GameConstants.PLATE_VALUE;
```
Comment (Ishadijcks) [inline src/scripts/underground/Underground.ts]: Watch out, here you increment the variable directly. This means that statistics aren't logged and possible bonuses aren't applied. Take a look at `App.game.shards.gainShards()` :)

Comment (nathan-c-jones) [reply src/scripts/underground/Underground.ts]: I missed that function somehow, change has been made!

**Pull request merged**